test/sdk-benchmarks/src/main/java/software/amazon/awssdk/benchmark/BenchmarkRunner.java

```diff
@@ -74,6 +83,7 @@ public static void main(String... args) throws RunnerException, JsonProcessingEx
         benchmarksToRun.addAll(ASYNC_BENCHMARKS);
         benchmarksToRun.addAll(PROTOCOL_BENCHMARKS);
         benchmarksToRun.addAll(COLD_START_BENCHMARKS);
+        benchmarksToRun.addAll(MAPPER_BENCHMARKS);
```

Comment:
Should we add ddb enhanced client data to `baseline.json` for regression detection?

Reply:
See previous comment for getting baseline